test/test.py

```diff
@@ -271,7 +271,7 @@ def generate_value(path):
         for obj in session.bulk.get(iterator, return_data_objects=True,
                                     verbose=True):
             counter += 1
-            key = attribute.title()
+            key = attribute.title().lower()
```

Comment:
I'm a bit confused here -- could you elaborate a bit? E.g.:
- can you help me understand why was ".title()" needed originally? (assuming it was really needed)
- does your modification (which basically equals "attribute.lower()") work in case-sensitive databases with attributes that are not strictly lower case? (e.g. if we would set attribute ="Kind" instead of 'kind' in this test).

Reply:
* The original need for ".title()" might be related to the previous (case-insensitive) settings of the MySQL databases, but it's not really clear. In any case, it is not needed anymore now.

* The test will be expanded to cover non-lowercase attributes in a future PR.